bundles/org.openhab.binding.miio/README.md

```diff
@@ -1131,6 +1143,45 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | current          | Number  | Current                             |
 | temperature      | Number  | Temperature                         |
 
+### Vacuum 1C STYTJ01ZHM (<a name="dreame-vacuum-mc1808">dreame.vacuum.mc1808</a>) Channels
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| BatteryLevel     | Number  | Battery-Battery Level               |
+| ChargingState    | Number  | Battery-Charging State              |
+| FirmwareRevision | String  | Device Information-Current Firmware Version |
+| Manufacturer     | String  | Device Information-Device Manufacturer |
+| Model            | String  | Device Information-Device Model     |
+| SerialNumber     | String  | Device Information-Device Serial Number |
+| Fault            | Number  | Robot Cleaner-Device Fault          |
+| Status           | Number  | Robot Cleaner-Status                |
+| BrushLeftTime    | String  | Main Cleaning Brush-Brush Left Time |
+| BrushLifeLevel   | Number  | Main Cleaning Brush-Brush Life Level |
+| FilterLifeLevel  | Number  | Filter-Filter Life Level            |
+| FilterLeftTime   | String  | Filter-Filter Left Time             |
+| BrushLeftTime1   | String  | Side Cleaning Brush-Brush Left Time |
+| BrushLifeLevel1  | Number  | Side Cleaning Brush-Brush Life Level |
+| WorkMode         | Number  | clean-workmode                      |
+| Area             | String  | clean-area                          |
+| Timer            | String  | clean-timer                         |
+| Mode             | Number  | clean-mode                          |
+| TotalCleanTime   | String  | clean-total time                    |
+| TotalCleanTimes  | String  | clean-total times                   |
+| TotalCleanArea   | String  | clean-                              |
+| CleanLogStartTime | String  | clean-                              |
+| ButtonLed        | String  | clean-led                           |
+| TaskDone         | Number  | clean-task done                     |
+| LifeSieve        | String  | consumable-life-sieve               |
+| LifeBrushSide    | String  | consumable-life-brush-side          |
+| LifeBrushMain    | String  | consumable-life-brush-main          |
```

Comment:
Are these supposed to be percentage values?

Reply:
see the long comment below